### PR TITLE
Add guidelines for node PR triage

### PR DESCRIPTION
**Pull request by ehashman**
**Which issue(s) this PR fixes**:

Adds some guidelines for SIG Node PR Triage on the project board: https://github.com/orgs/kubernetes/projects/49

so more contributors can get involved!

/hold
for discussion

/cc @SergeyKanzhelev

### CODE REVIEW

File: contributors/devel/sig-node/triage.md
Comment (knabben) [inline contributors/devel/sig-node/triage.md:21]: could you make explicit the prow add them by default when the PR is opened?

File: contributors/devel/sig-node/triage.md
Comment (knabben) [inline contributors/devel/sig-node/triage.md:72]: who can `/triage accepted`? It can be worth clarifying the roles suitable to do the procedure.

File: contributors/devel/sig-node/triage.md
Comment (dims) [inline contributors/devel/sig-node/triage.md:14]: wanna add a link to the handy bot commands too? - https://go.k8s.io/bot-commands

File: contributors/devel/sig-node/triage.md
Comment (dims) [inline contributors/devel/sig-node/triage.md:48]: `critical-urgen` also tells the release team that they can't make a release.

File: contributors/devel/sig-node/triage.md
Comment (dims) [reply contributors/devel/sig-node/triage.md:72]: today we don't have special roles in sig-node other than the sig-chair and TL. So for now i'd say any regular contributor to sig-node should be able to do this.

File: contributors/devel/sig-node/triage.md
Comment (dims) [inline contributors/devel/sig-node/triage.md:110]: files->directories.

File: contributors/devel/sig-node/triage.md
Comment (kikisdeliveryservice) [inline contributors/devel/sig-node/triage.md:82]: Note: there's something fussy that happens when there _is_ a release note but the bot doesn't pick it up and the author has to try to fix the text in their description - i've recently seen authors try to update their notes and bot still doesn't pick it up (also can't be manually overridden to add `release-note`).

File: contributors/devel/sig-node/triage.md
Comment (kikisdeliveryservice) [reply contributors/devel/sig-node/triage.md:48]: if it's critical-urgent feels like maybe we should do something more than just tag it? maybe also post in sig-node channel to flag?

File: contributors/devel/sig-node/triage.md
Comment (kikisdeliveryservice) [reply contributors/devel/sig-node/triage.md:82]: I guess what Im saying is that if it only has dnm/release-note-needed it's prob not sufficient to stay in this column?

**Review by kikisdeliveryservice — COMMENTED** [review]
I've been doing some triaging lately so added a couple notes. This is really helpful!!!!

File: contributors/devel/sig-node/triage.md
Comment (ehashman) [reply contributors/devel/sig-node/triage.md:72]: Technically anyone who's a Kubernetes org member can apply the label. I've definitely done some drive-by triage for other SIGs :)

File: contributors/devel/sig-node/triage.md
Comment (ehashman) [reply contributors/devel/sig-node/triage.md:82]: If we can remove the label by setting `/release-note-none`, as long as there's no other issues it can move. If not, it requires action from the author to fix the release note. I'll try to make that explicit.

**Comment by ehashman** [comment]
I addressed feedback in a separate commit so you can view the diff, I will squash later on request. cc @pacoxu @dims @kikisdeliveryservice @knabben

File: contributors/devel/sig-node/triage.md
Comment (SergeyKanzhelev) [inline contributors/devel/sig-node/triage.md:43]:
```suggestion
If the PR is trivial and doesn't provide much value, feel free to close it (using `/close` command) and
```

File: contributors/devel/sig-node/triage.md
Comment (SergeyKanzhelev) [inline contributors/devel/sig-node/triage.md:47]:
```suggestion
using the `/close` command and link the author to the [trivial edits policy].

[Kubernetes org members]: https://github.com/kubernetes/community/blob/master/community-membership.md#member
[trivial edits policy]: https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits

>  `/close`
>
> `Thank you for your PR. It seems to only contain trivial edits. Please read our [trivial edits policy](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits) and reopen with the description if you believe that this PR is needed.`
```

File: contributors/devel/sig-node/triage.md
Comment (SergeyKanzhelev) [inline contributors/devel/sig-node/triage.md:63]: Maybe have backlog before important-longeterm to demonstrate the order of priorities

Comment (SergeyKanzhelev) [reply contributors/devel/sig-node/triage.md:63]: according to https://github.com/kubernetes/test-infra/blob/master/label_sync/labels.md

File: contributors/devel/sig-node/triage.md
Comment (SergeyKanzhelev) [inline contributors/devel/sig-node/triage.md:64]: Also I'd go with other examples:

```suggestion
- **priority/backlog:** Non-urgent changes such as minor performance optimizations, improving error logs, increasing tests code coverage, code refactoring, and addressing code analysis issues.
```

File: contributors/devel/sig-node/triage.md
Comment (SergeyKanzhelev) [inline contributors/devel/sig-node/triage.md:79]: Also mention to check the release note accuracy. It is often forgotten

File: contributors/devel/sig-node/triage.md
Comment (SergeyKanzhelev) [inline contributors/devel/sig-node/triage.md:89]:
```suggestion
reviewer may have requested changes, tests might be failing, or a PR may have one of the following
```

File: contributors/devel/sig-node/triage.md
Comment (SergeyKanzhelev) [inline contributors/devel/sig-node/triage.md:101]:
```suggestion
Author of the PR is encouraged to resolve or reply to all the comments and comment on PR to indicate that is ready for the next iteration of reviews.

PRs that do not have any of the above labeled in this column will be
evaluated occasionally to see if they are ready for review.

If left untouched for 30 days, PR will be marked as rotten and will eventually be closed.
```

File: contributors/devel/sig-node/triage.md
Comment (SergeyKanzhelev) [inline contributors/devel/sig-node/triage.md:120]: what do you envision to have here?

File: contributors/devel/sig-node/triage.md
Comment (SergeyKanzhelev) [inline contributors/devel/sig-node/triage.md:139]: can you copy/paste an example here? Might be helpful

File: contributors/devel/sig-node/triage.md
Comment (SergeyKanzhelev) [inline contributors/devel/sig-node/triage.md:151]: will this column contain PRs that are waiting for approval from other OWNERS, API review, or release managers for cherry-picks?

File: contributors/devel/sig-node/triage.md
Comment (SergeyKanzhelev) [inline contributors/devel/sig-node/triage.md:144]: I wouldn't encourage anyone to approve. It creates noise in PR comments.

File: contributors/devel/sig-node/triage.md
Comment (SergeyKanzhelev) [inline contributors/devel/sig-node/triage.md:134]:
```suggestion
- lgtm (during triage some lgtm's from people not directly involved with SIG Node may not be enough for approver to look at the PR)
```

File: contributors/devel/sig-node/triage.md
Comment (ehashman) [reply contributors/devel/sig-node/triage.md:63]: These should be in order already? critical-urgent > important-soon > important-longterm > backlog > awaiting-more-evidence (which I didn't include)

File: contributors/devel/sig-node/triage.md
Comment (ehashman) [reply contributors/devel/sig-node/triage.md:79]: I will add that to the review section; I don't think it's necessary as part of triage.

File: contributors/devel/sig-node/triage.md
Comment (ehashman) [reply contributors/devel/sig-node/triage.md:89]: I typically do *not* put PRs into **Waiting on Author** if tests are failing because tests are rather flaky, only if there is a confirmed test failure that needs to be addressed (and usually at that point I put a hold on it).

File: contributors/devel/sig-node/triage.md
Comment (ehashman) [reply contributors/devel/sig-node/triage.md:120]: More node specific content on reviewing, code structure, etc. for components we control would be nice.

File: contributors/devel/sig-node/triage.md
Comment (ehashman) [reply contributors/devel/sig-node/triage.md:134]: I'll add a note about that outside of this list.

File: contributors/devel/sig-node/triage.md
Comment (ehashman) [reply contributors/devel/sig-node/triage.md:144]: I don't disagree but people do it :smile: I don't think this is encouraging it, but I can rewrite.

File: contributors/devel/sig-node/triage.md
Comment (SergeyKanzhelev) [reply contributors/devel/sig-node/triage.md:89]: Maybe we should to save reviewers time. It's easier to review when you know that tests are passing. Maybe link instruction on how to investigate test failures.

File: contributors/devel/sig-node/triage.md
Comment (ehashman) [reply contributors/devel/sig-node/triage.md:143]: (Not in the scope of this PR so I'm going to close this)

File: contributors/devel/sig-node/triage.md
Comment (SergeyKanzhelev) [reply contributors/devel/sig-node/triage.md]: My understanding before was that backlog is higher priority to look at as its something that can be looked at immediately. Important longterm may not even be clear on design. But I agree with this ordering, thank you for clarification!

**Comment by kikisdeliveryservice** [comment]
One q: if the `kind` flag is required in some of your columns, do we need to have those options explained as well?

**Review by pacoxu — APPROVED** [review]

**Comment by kikisdeliveryservice** [comment]
I :heart: this doc so much, thank you @ehashman for all of your hard work!!!!

/lgtm

**Comment by ehashman** [comment]
/hold cancel

will leave this for an approver to look over and merge

**Review by SergeyKanzhelev — APPROVED** [review]
/lgtm

**Comment by ehashman** [comment]
/assign @dchen1107 

can you PTAL and approve the new documentation for the Node triage process? :)

**Comment by derekwaynecarr** [comment]
@ehashman this looks great.

/approve
/lgtm

**Pull request merged**
